docs/src/docs/asciidoc/index.adoc

```diff
@@ -22,6 +22,8 @@ If you are using alternative build systems, see <<alternative-build-systems.adoc
 === Release 0.10.3
 
 - Remove usage of macro from merger tool initialization and throw better error if executable does not exist
+- Adjust Native Build Tools to the reachability-metadata.json
```

Comment:
I don't know what this means, could you make it clearer what this means for users?